### PR TITLE
Merge updates into v1

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+-->
+## What
+<!-- A clear description of what the problem or feature request is -->
+
+## Details
+<!--
+If it is a bug:
+- describe what the expected behaviour is, and what is actually happening
+- add steps on how to reproduce it
+- describe the environment you are having this problem in
```
Comment (notlee) [inline .github/ISSUE_TEMPLATE.md:19]: maybe we should update the issue template to be more relevant to the project board -- remove references to browser versions, etc?

File: README.md [hunk 2]
```diff
@@ -7,10 +7,25 @@ Automatically add all new issues and pull requests to the Origami Project Board!
 See [./example.yml](./example.yml)
 
 ### 1. Adding the secret
-1. Get the secret from LastPass. It's in a Shared-Origami note called `Origami fox issue and pull request workflow secret`.
+
+#### Manual approach:
+
+1. Get the secret from [Vault](https://vault.in.ft.com:8080/ui/vault/secrets/secret/show/teams/origami/github.com/origamiserviceuser). It is named `ORIGAMI_FOX_ACCESS_TOKEN`.
```
Comment (notlee) [inline README.md:13]: Might be worth updating this to note these secrets are available to the org now, and should not need adding but can be found in vault?

File: add-secret-to-repo.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
+"use strict"
+
+process.on("unhandledRejection", error => {
```
Comment (notlee) [inline add-secret-to-repo.js:3]: do we need this script with the org wide envs?

File: README.md (hunk 2 above)
Comment (JakeChampion) [reply README.md]: I updated it, good spot!

**Pull request merged**